.idea/gradle.xml

```diff
@@ -9,6 +9,7 @@
           <set>
             <option value="$PROJECT_DIR$" />
             <option value="$PROJECT_DIR$/app" />
+            <option value="$PROJECT_DIR$/measurement_kit" />
```

Comment:
@AntonioLangiu I think this may be the reason why we see a spurious measurement_kit directory, isn't it?